### DISCUSSION TITLE
Use Valent in conjunction with kdeconnect/gsconnect?

### DISCUSSION THREAD

**Discussion by edgar-vincent** [andyholmes/valent#240] [Help] [answered]
Hi!

Thanks a lot for developing this! 
Please excuse me if my question is silly, but how compatible is valent with kdeconnect/gsconnect? Can valent connect to kdeconnect? Can valent and kdeconnect run simultaneously?

Thanks again!

EV

**Comment by andyholmes** [comment, marked as answer]
Hi!

Valent aims to be compatible with KDE Connect (and GSConnect), so it should be able to connect to both. I still run KDE Connect on my Android phone and it works well.

The KDE Connect protocol defaults to discovering devices on port 1716, so you still can't two clients on the same machine. Valent will not popup a notification if it can't own the port, so if you start it with another client you should exit it and then toggle the LAN plugin on/off (`Main Window => Preferences`):

![Screenshot from 2022-08-22 10-49-10](https://user-images.githubusercontent.com/1265208/185986523-f75ab997-8400-47a4-83e5-9b4b9c849cf7.png)

**Comment by edgar-vincent** [comment]
Thank you very much for your reply.
I did manage to connect gsconnect (on my laptop) to valent (on my Pinephone) by entering the phone's IP manually in gsconnect. Auto-discovery didn't seem to work, so I thought the two weren't compatible. 
If this is not an issue you are aware of, I'd be happy to provide debug information.

Thanks again!

EV

**Reply by andyholmes** [reply]
That should work, I think I tested it in the past. I'm out for couple weeks housesitting so I don't have my PinePhone, but there may just be some sleep-mode or broadcast quirks with that.
